lib/core/src/lib/form/components/middlewares/decimal-middleware.service.spec.ts

```diff
@@ -0,0 +1,83 @@
+/*!
+ * @license
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { FormFieldModel, FormFieldTypes, FormModel } from '../widgets';
+import { DecimalRenderMiddlewareService } from './decimal-middleware.service';
+
+describe('DecimalRenderMiddlewareService', () => {
+    let decimalMiddlewareService: DecimalRenderMiddlewareService;
+    let formFieldModel: FormFieldModel;
+
+    beforeEach(() => {
+        decimalMiddlewareService = new DecimalRenderMiddlewareService();
+
+        const form = new FormModel();
+        formFieldModel = new FormFieldModel(form, {
+            type: FormFieldTypes.DECIMAL,
+            id: 'id',
+            precision: 3,
+            value: '10.1060'
+        });
+    });
+
+    it('should return field with proepr precisison', () => {
+        formFieldModel.value = '10.1000000000000';
+        formFieldModel.precision = 3;
+        const parsedField = decimalMiddlewareService.getParsedField(formFieldModel);
+        expect(parsedField.value).toBe('10.100');
+    });
+
+    it('should round up number with correct precisison', () => {
+        formFieldModel.value = '10.1039999';
+        formFieldModel.precision = 3;
+        const parsedField = decimalMiddlewareService.getParsedField(formFieldModel);
+        expect(parsedField.value).toBe('10.104');
+    });
+
+    it('should round up number, when removed fraction part starts with number 5 ', () => {
```

Comment:
```suggestion
    it('should round up number, when removed fraction part starts with number larger than 4 ', () => {
```

Reply:
changed to `when removed fraction part starts with number larger or equal 5`

Want to emphasis the boundary, which is 5 here :)